docs/usage.rst

```diff
@@ -33,180 +6,152 @@ a SQLite_ database.
     import audeer
 
     _cwd_root = os.getcwd()
-    _tmp_root = audeer.mkdir(os.path.join('docs', 'tmp'))
+    _tmp_root = audeer.mkdir('docs', 'tmp-usage')
     os.chdir(_tmp_root)
 
 
-.. _file-system-example:
-
-File-system example
--------------------
+.. _usage:
 
-The class
-:class:`audbackend.backend.FileSystem`
-implements
-:class:`audbackend.backend.Base`
-for a standard file system.
+Usage
+=====
 
+With the help of :mod:`audbackend`
+a user can store files
+in a repository
+on a storage system.
 
-We start by registering the class
-(in fact,
-the class is registered
-by default,
-but it doesn't hurt
-if we do it again).
+The user can select between different :ref:`interfaces <interfaces>`
+that influence how the data is structured
+and presented to the user.
+In addition,
+:mod:`audbackend` supports different storage systems,
+so called :ref:`backends <backends>`.
+The names of all available backends can be listed by:
 
 .. jupyter-execute::
 
     import audbackend
 
-    audbackend.register('file-system', audbackend.backend.FileSystem)
+    list(audbackend.available())
+
 
+.. _unversioned-data-on-a-file-system:
 
-To make sure we can keep track
-of all existing backend instances,
-we use :func:`audbackend.create`
-to create a repository
-instead of calling the class ourselves.
-We provide three arguments:
+Unversioned data on a file system
+---------------------------------
 
-* ``name``: the name under which the backend class is registered
-* ``host``: the host address,
-  in this case a folder on the local file system.
-* ``repository``: the repository name,
-  in this case a sub-folder within the ``host`` folder
-  (it is possible to have several repositories
-  on the same host).
+As all data is stored in a repository
+on the backend,
+we need to first create one.
+We select the ``'file-system'`` backend.
 
 .. jupyter-execute::
     :hide-output:
 
     audbackend.create('file-system', './host', 'repo')
 
-
-This will create an empty repository
-(in our case the folder ``'./host/repo/'``).
-To view all available instances,
-we would do:
-
-.. jupyter-execute::
-
-    audbackend.available()
-
-
-We can access an existing repository with:
+Once we have an existing repository,
+we can access it with :func:`audbackend.access`.
+We choose the :class:`audbackend.interface.Unversioned` interface
+to communicate with the repository,
+instead of the default one
+(:class:`audbackend.interface.Versioned`).
```

Comment:
Is it important to mention the default here? Though it's probably obvious I think we should explain what unversioned access means, e.g. "We use the Unversioned interface. It does not support versioning, i.e. exactly one file exists for a backend path."

Reply:
I adjusted the text accordingly.